src/services/teams/teamService.ts

```diff
@@ -1,6 +1,7 @@
 import { Repository } from "typeorm";
 import { Team, User } from "../../db/entity/hub";
 import { UserService } from "../users";
+import { ApiError, HttpResponseCode } from "../../util/errorHandling";
```

Comment:
Do these actually get used anywhere in this file?

Reply:
Nice catch, they aren't used in this service